contrib/php/template.py

```diff
@@ -0,0 +1,276 @@
+    "--datarootdir=/usr/share/php",
+    "--includedir=/usr/include",
+    "--libdir=/usr/lib/php",
+    "--localstatedir=/var",
+    "--mandir=/usr/share/man",
+    "--prefix=/usr",
+    "--program-prefix=",
+    "--program-suffix=",
+    "--sbindir=/usr/bin",
+    "--sysconfdir=/etc/php",
+    "--with-config-file-path=/etc/php",
+    "--with-config-file-scan-dir=/etc/php/conf.d",
+    "--with-layout=GNU",
+    # php-fpm
+    "--enable-fpm",
+    "--with-fpm-group=_php",
+    "--with-fpm-user=_php",
+    # extensions
+    "--disable-all",
+    "--enable-bcmath=shared",
+    "--with-bz2=shared",
+    "--enable-calendar=shared",
+    "--enable-ctype=shared",
+    "--with-curl=shared",
+    "--enable-dom=shared",
+    "--enable-exif=shared",
+    "--with-ffi=shared",
+    "--enable-fileinfo=shared",
+    "--enable-filter=shared",
+    "--enable-ftp=shared",
+    "--enable-gd=shared",
+    "--with-external-gd",
+    "--with-gettext=shared",
+    "--with-gmp=shared",
+    "--with-iconv=shared",
+    "--enable-intl=shared",
+    "--with-libedit=shared",
+    "--with-libxml",
+    "--enable-mbstring=shared",
+    "--with-mhash",
+    "--enable-opcache=shared",
+    "--with-openssl=shared",
+    "--with-password-argon2",
+    "--enable-pcntl=shared",
+    "--enable-phar=shared",
+    "--enable-posix=shared",
+    "--enable-session=shared",
+    "--enable-shmop=shared",
+    "--enable-simplexml=shared",
+    "--enable-soap=shared",
+    "--enable-sockets=shared",
+    "--with-sodium=shared",
+    "--with-sqlite3=shared",
+    "--enable-sysvmsg=shared",
+    "--enable-sysvsem=shared",
+    "--enable-sysvshm=shared",
+    "--enable-tokenizer=shared",
+    "--with-unixODBC=shared",
+    "--enable-xml=shared",
+    "--enable-xmlreader=shared",
+    "--enable-xmlwriter=shared",
+    "--with-xsl=shared",
+    "--with-zip=shared",
+    "--with-zlib=shared",
+]
+configure_gen = []
+make_check_target = "test"
+make_check_env = {
+    "LANG": "",
+    "LC_ALL": "",
+    "NO_INTERACTION": "1",
+    "REPORT_EXIT_STATUS": "1",
+    "SKIP_IO_CAPTURE_TESTS": "1",
+    "SKIP_ONLINE_TESTS": "1",
+    "SKIP_PERF_SENSITIVE": "1",
+    "SKIP_SLOW_TESTS": "1",
+    "TEST_TIMEOUT": "10",
+    "TZ": "",
+}
+hostmakedepends = [
+    "automake",
+    "bison",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "argon2-devel",
+    "freetype-devel",
+    "gettext-devel",
+    "gmp-devel",
+    "icu-devel",
+    "libcurl-devel",
+    "libedit-devel",
+    "libffi-devel",
+    "libgd-devel",
+    "libjpeg-turbo-devel",
+    "libsodium-devel",
+    "libxml2-devel",
+    "libxslt-devel",
+    "libzip-devel",
+    "oniguruma-devel",
+    "openssl-devel",
+    "sqlite-devel",
+    "unixodbc-devel",
+    "zlib-devel",
+]
+checkdepends = ["procps"]
+pkgdesc = "HTML-embedded scripting language"
+maintainer = "Renato Botelho do Couto <renato@netgate.com>"
+license = "PHP-3.01"
+url = "https://www.php.net"
+source = f"{url}/distributions/php-{pkgver}.tar.xz"
```

Comment:
tar.gz

Reply:
that's just worse than the xz one, there's no point to use the gz tarball

Reply:
i'm pretty sure nobody is deciding to unilaterally do that and it's a huge overreaction